smb_lowlevel_controller/smb_lowlevel_controller/src/SmbHWInterface.cpp

```diff
@@ -260,16 +262,20 @@ bool reglimits = ((urdf_limits_ok && urdf_soft_limits_ok) || (rosparam_limits_ok
     }
 
 
-    WheelVelocityControl::WheelVelocityControl(double* const setPoint, double* const command, const double * const processValue) :
-     setPoint_(setPoint),  command_(command),  processValue_(processValue), loop_count_(0)
+    WheelVelocityControl::WheelVelocityControl(double* const setPoint, double* const otherSetPoint, double* const command, const double * const processValue) :
+     setPoint_(setPoint), otherSetPoint_(otherSetPoint),  command_(command),  processValue_(processValue), loop_count_(0)
     {}
 
     WheelVelocityControl::~WheelVelocityControl() {delete controller_state_publisher_;};
 
-    bool WheelVelocityControl::init(ros::NodeHandle& nh, const std::string& nhprefix, bool publishControllerState) {
+    bool WheelVelocityControl::init(ros::NodeHandle& nh, ros::NodeHandle& private_nh, const std::string& nhprefix, bool publishControllerState) {
       if (publishControllerState) {
         controller_state_publisher_ = new realtime_tools::RealtimePublisher<control_msgs::JointControllerState>(nh, "iPID_state", 1);
       }
+
+      private_nh.param<double>("/control/lowlevel_controller/WHEEL_JOINT_ff_param/ff_general", ff_general_, 1.5);
```

Comment:
here, the name space of the parameter is hard coded. 
by using the private nodehandle, you should be able to use a relative parameter name.